scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p><s>This</s> <strong>The goal of this</strong> notebook is to evaluate <a href="https://github.com/ifzhang/FairMOT" target="_blank">FairMOT</a> <s>with</s><strong> on the</strong> <a href="https://motchallenge.net/" target="_blank">MOT Challenge dataset</a>. <s>It hosts the most</s> <strong>This is a</strong> <s>common</s> <strong>popular</strong> benchmarking dataset<s>s</s> for <s>pedestrian MOT</s> <strong>measuring multi-object tracking performance on pedestrian data</strong>. <s>Different datasets exist:</s> <strong>The collection includes </strong>MOT15, MOT16/17, <strong>and </strong>MOT 19/20. These datasets contain many video sequences<s>,</s> with different tracking difficulty levels<s>, with annotated</s> <strong>and</strong> ground-truth <strong>annotations</strong>. Detections are also provided for optional use<s> by the participating tracking algorithms</s>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>